platform/android/export/export.cpp

```diff
@@ -2299,6 +2352,8 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 					return ERR_UNCONFIGURED;
 				}
 			}
+			// instead of calling _fix_resources
+			_create_project_name_strings_files(p_preset);
```

Comment:
This should be behind a flag, and you're missing the follow-up logic to disable `_fix_resources` for the app name if this is enabled.

Reply:
This line of code is inside of a block that's behind a `use_custom_build` flag. I pushed another commit just now that disables the future call to `_fix_resources` if `use_custom_build` is true.